lib/note-detail/index.tsx

```diff
@@ -241,6 +245,7 @@ export class NoteDetail extends Component {
 }
 
 const mapStateToProps = ({ appState: state, ui, settings }) => ({
+  detectLanguage: settings.languageDetectionEnabled,
```

Comment:
we might as well add this inside of `<NoteContentEditor />` so we don't have to pass it _through_ this component.

Reply:
updated in ef9fcb7